README.md

```diff
@@ -74,6 +78,12 @@ $ pip install -r requirements.txt
 $ make test
 ```
 
+### For SUSE distributions packages are available
```

Comment:
Can you move this above the `On other Linux distributions` section so the doc reads like:
```
- AML
- SUSE
- On other Linux distributions
```

This makes it nicer to add additional sections in the future for distributed packages.